osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/AlarmHelperService.java

```diff
@@ -1,20 +1,21 @@
 /**
  * Copyright 2015 Smart Society Services B.V.
  *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
  */
-package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands;
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.alarm;
 
 import java.util.Collections;
 import java.util.EnumMap;
 import java.util.Map;
 import java.util.Set;
 
-import org.springframework.stereotype.Service;
-
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils.ByteRegisterConverter;
 import org.opensmartgridplatform.dto.valueobjects.smartmetering.AlarmTypeDto;
+import org.springframework.stereotype.Service;
 
 @Service("alarmHelperService")
 public class AlarmHelperService {
```

Comment:
Note that the class AlarmHelperService is also used by `org.opensmartgridplatform.adapter.protocol.dlms.infra.networking.DlmsPushNotificationDecoder`. So maybe the packaging is not 100% logical yet?

Also a couple of the methods are only used in tests. I would check if these can be removed, or extracted into a test utility...